src/types/transaction.zig

```diff
@@ -145,12 +217,37 @@ test "Transaction basics" {
         try tx.addOutput(50000, script_pubkey);
     }
 
-    try testing.expectEqual(@as(usize, 1), tx.inputs.items.len);
-    try testing.expectEqual(@as(usize, 1), tx.outputs.items.len);
-    try testing.expectEqual(@as(i64, 50000), tx.outputs.items[0].value);
+    try testing.expectEqual(@as(usize, 1), tx.inputs.len);
+    try testing.expectEqual(@as(usize, 1), tx.outputs.len);
+    try testing.expectEqual(@as(i64, 50000), tx.outputs[0].value);
```

Comment:
all those `@as` are unecessary

Reply:
It was a test of Transaction's basic, I did not work on this, this test was present before I extended Transaction's features. Should I remove this @as ?

Reply:
yep, please. It's always good to improve the codebase even if it wasn't your code